[2023-04-14 16:17:00,719] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: vic_investigacion_ETL.get_cv_lac_data scheduled__2023-04-14T00:00:00+00:00 [queued]>
[2023-04-14 16:17:00,726] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: vic_investigacion_ETL.get_cv_lac_data scheduled__2023-04-14T00:00:00+00:00 [queued]>
[2023-04-14 16:17:00,726] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-04-14 16:17:00,727] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-04-14 16:17:00,727] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-04-14 16:17:00,737] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): get_cv_lac_data> on 2023-04-14 00:00:00+00:00
[2023-04-14 16:17:00,752] {standard_task_runner.py:52} INFO - Started process 257 to run task
[2023-04-14 16:17:00,758] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'vic_investigacion_ETL', 'get_cv_lac_data', 'scheduled__2023-04-14T00:00:00+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/vic_investigacion_dag.py', '--cfg-path', '/tmp/tmpdwfq69x3', '--error-file', '/tmp/tmppbinhupv']
[2023-04-14 16:17:00,761] {standard_task_runner.py:80} INFO - Job 48: Subtask get_cv_lac_data
[2023-04-14 16:17:00,858] {task_command.py:371} INFO - Running <TaskInstance: vic_investigacion_ETL.get_cv_lac_data scheduled__2023-04-14T00:00:00+00:00 [running]> on host c91627f658c0
[2023-04-14 16:17:00,945] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=vic_investigacion_ETL
AIRFLOW_CTX_TASK_ID=get_cv_lac_data
AIRFLOW_CTX_EXECUTION_DATE=2023-04-14T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-14T00:00:00+00:00
[2023-04-14 16:17:00,947] {logging_mixin.py:115} INFO - Getting CV_LAC Data
[2023-04-14 16:17:00,947] {python.py:173} INFO - Done. Returned value was: None
[2023-04-14 16:17:00,962] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=vic_investigacion_ETL, task_id=get_cv_lac_data, execution_date=20230414T000000, start_date=20230414T161700, end_date=20230414T161700
[2023-04-14 16:17:01,010] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-04-14 16:17:01,048] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-04-14 16:39:37,547] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: vic_investigacion_ETL.get_cv_lac_data scheduled__2023-04-14T00:00:00+00:00 [queued]>
[2023-04-14 16:39:37,554] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: vic_investigacion_ETL.get_cv_lac_data scheduled__2023-04-14T00:00:00+00:00 [queued]>
[2023-04-14 16:39:37,555] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-04-14 16:39:37,555] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-04-14 16:39:37,556] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-04-14 16:39:37,565] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): get_cv_lac_data> on 2023-04-14 00:00:00+00:00
[2023-04-14 16:39:37,571] {standard_task_runner.py:52} INFO - Started process 95 to run task
[2023-04-14 16:39:37,576] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'vic_investigacion_ETL', 'get_cv_lac_data', 'scheduled__2023-04-14T00:00:00+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/vic_investigacion_dag.py', '--cfg-path', '/tmp/tmpxbbage0d', '--error-file', '/tmp/tmpdvzkexn0']
[2023-04-14 16:39:37,579] {standard_task_runner.py:80} INFO - Job 55: Subtask get_cv_lac_data
[2023-04-14 16:39:37,675] {task_command.py:371} INFO - Running <TaskInstance: vic_investigacion_ETL.get_cv_lac_data scheduled__2023-04-14T00:00:00+00:00 [running]> on host d1b5acf00b55
[2023-04-14 16:39:37,750] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=vic_investigacion_ETL
AIRFLOW_CTX_TASK_ID=get_cv_lac_data
AIRFLOW_CTX_EXECUTION_DATE=2023-04-14T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-14T00:00:00+00:00
[2023-04-14 16:39:37,753] {logging_mixin.py:115} INFO - Getting CV_LAC Data
[2023-04-14 16:39:37,754] {python.py:173} INFO - Done. Returned value was: None
[2023-04-14 16:39:37,769] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=vic_investigacion_ETL, task_id=get_cv_lac_data, execution_date=20230414T000000, start_date=20230414T163937, end_date=20230414T163937
[2023-04-14 16:39:37,827] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-04-14 16:39:37,854] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-05-31 23:29:31,548] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: vic_investigacion_ETL.get_cv_lac_data scheduled__2023-04-14T00:00:00+00:00 [queued]>
[2023-05-31 23:29:31,555] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: vic_investigacion_ETL.get_cv_lac_data scheduled__2023-04-14T00:00:00+00:00 [queued]>
[2023-05-31 23:29:31,556] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-05-31 23:29:31,556] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-05-31 23:29:31,557] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-05-31 23:29:31,564] {taskinstance.py:1397} INFO - Executing <Task(ReadCvLAC): get_cv_lac_data> on 2023-04-14 00:00:00+00:00
[2023-05-31 23:29:31,569] {standard_task_runner.py:52} INFO - Started process 122 to run task
[2023-05-31 23:29:31,574] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'vic_investigacion_ETL', 'get_cv_lac_data', 'scheduled__2023-04-14T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/vic_investigacion_dag.py', '--cfg-path', '/tmp/tmpozjtv8fu', '--error-file', '/tmp/tmpbqv8f9zn']
[2023-05-31 23:29:31,576] {standard_task_runner.py:80} INFO - Job 4: Subtask get_cv_lac_data
[2023-05-31 23:29:31,648] {task_command.py:371} INFO - Running <TaskInstance: vic_investigacion_ETL.get_cv_lac_data scheduled__2023-04-14T00:00:00+00:00 [running]> on host 5d9ccb732815
[2023-05-31 23:29:31,735] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=vic_investigacion_ETL
AIRFLOW_CTX_TASK_ID=get_cv_lac_data
AIRFLOW_CTX_EXECUTION_DATE=2023-04-14T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-14T00:00:00+00:00
[2023-05-31 23:29:31,740] {logging_mixin.py:115} INFO - Getting CV LAC Data
[2023-05-31 23:29:31,756] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=vic_investigacion_ETL, task_id=get_cv_lac_data, execution_date=20230414T000000, start_date=20230531T232931, end_date=20230531T232931
[2023-05-31 23:29:32,111] {local_task_job.py:156} INFO - Task exited with return code 0
